packages/mobile/src/send/SendAmount.tsx

```diff
@@ -155,9 +160,23 @@ export class SendAmount extends React.Component<Props, State> {
     this.props.fetchPhoneAddresses([recipient.e164PhoneNumber])
   }
 
+  getDollarsAmount = () => {
+    const parsedInputAmount = parseInputAmount(this.state.amount)
+
+    if (LOCAL_CURRENCY_SYMBOL) {
+      const { localCurrencyExchangeRate } = this.props
+      return (
+        convertLocalAmountToDollars(parsedInputAmount, localCurrencyExchangeRate) ||
+        new BigNumber('')
```

Comment:
why '' instead of 0 here?

Reply:
I wanted the equivalent to `NaN` if the exchange rate hasn't been fetched yet. 0 would be misleading.